auth-api/src/auth_api/resources/entity.py

```diff
@@ -135,6 +135,29 @@ def delete(business_identifier):
 
         return response, status
 
+@cors_preflight('GET,OPTIONS')
+@API.route('/<string:business_identifier>/authentication', methods=['GET', 'OPTIONS'])
```

Comment:
We should do the API changes in MAIN in a separate PR, because MAIN is deployed to DEV